examples/blockfrost-adapter-example.ts

```diff
@@ -33,6 +32,8 @@ import {
   StableswapCalculation,
   StableswapConstant,
 } from "../src";
+
+import { BlockfrostAdapter } from "../src/adapters/blockfrost";
```

Comment:
`src/adapters` should have `index.ts` file that export `Adapter` interface and the 3 adapters out
then `src/index.ts` should export the interface + 3 adapters as well
it makes sure lib consumers can `import { ... } from "@minswap/sdk"`

Reply:
done